### PR TITLE
fix configuration page to include configs based on supported config subsystems

### PR DESCRIPTION
**Pull request by prakashsvmx**
Fixes https://github.com/minio/console/issues/2945

Test in Single drive mode ( Heal option is not available)
and Erasure/Distributed mode. ( Heal option is present)

### CODE REVIEW

File: portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationOptions.tsx [hunk 1]
```diff
@@ -64,17 +66,56 @@ const getRoutePath = (path: string) => {
...
+              return acc;
+            },
+            []
+          );
+
+          setConfigSubSysList(confSubSysList);
+        }
+      })
+      .catch(() => {
+        console.log("Error in retrieving config subsystem list.");
```
[the remark this answers is not in the document]
Comment (prakashsvmx) [reply portal-ui/src/screens/Console/Configurations/ConfigurationPanels/ConfigurationOptions.tsx]: Thank you. i have updated to display the error to the user.

**Comment by prakashsvmx** [comment]
Thank you @harshavardhana :+1:  . I have addressed the review comments.

**Review by harshavardhana — DISMISSED** [review]

**Comment by harshavardhana** [comment]
```
[4/4] Building fresh packages...
success Saved lockfile.
Done in 13.70s.
yarn run v1.22.19
$ /home/runner/work/console/console/portal-ui/node_modules/.bin/prettier --check .
Checking formatting...
[warn] src/screens/Console/Configurations/ConfigurationPanels/ConfigurationOptions.tsx
[warn] Code style issues found in the above file. Run Prettier to fix.
error Command failed with exit code 1.
```

**Review by bexsoft — COMMENTED** [review]
Please update prettier to v3 & run `make pretty` under portal-ui, Thank you!

**Review by harshavardhana — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM, Detected an issue with override env variables not related to this PR. Will Fix accordingly in an upcoming PR

**Pull request merged**